src/pages/admin/index.tsx

```diff
@@ -1,5 +1,10 @@
 import React from 'react';
+import Sidebar from '@src/common/components/generics/Sidebar';
```

Comment:
Import this component from the index instead

Reply:
work done.